src/wazuh_modules/wm_exec.c

```diff
@@ -343,24 +349,46 @@ int wm_exec(char *command, char **output, int *exitcode, int secs)
 
             pthread_mutex_destroy(&tinfo.mutex);
             pthread_cond_destroy(&tinfo.finished);
+
+            // Wait for child process
```

Comment:
`waitpid()` must be called in any case, otherwise the child process will remain as zombie.